source/server-discovery-and-monitoring/server-discovery-and-monitoring-logging-and-monitoring.rst

```diff
@@ -251,6 +251,8 @@ Events that MUST be published (with their conditions) are as follows.
   /**
    * Fired when the server monitor's ``hello`` or legacy hello command is started - immediately before
    * the ``hello`` or legacy hello command is serialized into raw BSON and written to the socket.
+   * When the monitor is just starting, the first instance of this event is fired just after the
+   * socket is opened.
```

Comment:
This is also if the monitor is creating a new connection if the current one breaks. so approx ~ "if the monitor is creating a socket, this event is after the socket is connected"

We don't have this phrasing issue in the prose because we just mention "creating the socket" so I leave the exact wordsmithing to you.